frontend/src/lib/components/PropertyFilters/components/OperatorValueSelect.tsx

```diff
@@ -12,6 +12,7 @@ interface OperatorValueSelectProps {
     value?: string | number | Array<string | number> | null
     columnOptions?: ColProps | [ColProps, ColProps]
     placeholder?: string
+    valuesEndpoint?: string
```

Comment:
We now have a weird naming clash due to `valuesEndpoint` being a string in some places, a function in others.

Suggestion: Let's name this `endpoint` instead?